### ISSUE TITLE
Reapack won't run on Windows XP (SP3)

### ISSUE THREAD

**Issue by uncleswede** [cfillion/reapack#5]
I've downloaded Reapack onto my studio PC but get the error "The procedure entry point GetTickCount64 could not be located in the dynamic link library KERNEL32.DLL" when starting Reaper v5.211.

On investigation, it seems that the Windows XP version of Kernel32.dll does not have the GetTickCount64 function (this was introduced from Windows Vista onwards).

I installed Reapack on my mobile recording laptop (running Windows 8.1) and it worked fine with Reaper v5.211

So it appears that Reapack is *not* compatible with Windows XP :-(

**Comment by cfillion** [comment]
Thanks for the nice report!

XP support was theoretical as I was unable to find a version of Windows XP to test with...
But I got lucky today and finally found a [VM image](http://virtualization.modern.ie/vhd/IEKitV1_Final/VirtualBox/OSX/IE6_WinXP.zip) from Microsoft so I was able to make it work (yay!). Only to find that...

- Windows XP's implementation of SSL is utterly obsolete and cannot connect to any server using modern security settings (such as GitHub where all of ReaPack's default set of repositories are hosted).
  The solution is to add 1–2 MB to the ReaPack builds by including an alternate SSL implementation (namely, OpenSSL)
- I must also bundle a set of current [CA certificates](https://curl.haxx.se/docs/caextract.html) or disable remote server authentication altogether (meaning lessened security).
-  [XP itself](https://connect.microsoft.com/VisualStudio/Feedback/Details/1941836) and Microsoft's [C runtime](https://connect.microsoft.com/VisualStudio/feedback/details/1557168/wstat64-returns-1-on-xp-always) on XP have bugs which cause crashes and bad behavior.
- There are a few other glitches I haven't tried to debug yet

I'll pass for now and focus on Vista+ (haven't tested on that either, crossing fingers) but I'll keep the fix around just in case I change my mind in the future.
